forge-gui/res/cardsfolder/upcoming/call_forth_the_tempest.txt

```diff
@@ -0,0 +1,8 @@
+Name:Call Forth the Tempest
+ManaCost:5 R R R
+Types:Sorcery
+K:Cascade
+K:Cascade
+A:SP$ DamageAll | NumDmg$ X | ValidCards$ Creature.OppCtrl | ValidDescription$ each creature your opponents controls | SpellDescription$ CARDNAME deals damage to each creature your opponents control equal to the total mana value of other spells you've cast this turn.
+SVar:X:Count$ThisTurnCast_Card.YouCtrl+Other$SumCMC
```

Comment:
did you test that?

Reply:
Hmm you're right it only counts the amount of spells played (not the cmc)

Reply:
Any hints where I should look at